flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -95,7 +95,7 @@ public OperatorStateBackend createOperatorStateBackend(
 			Environment env,
 			String operatorIdentifier) throws Exception {
 
-		return new DefaultOperatorStateBackend(env.getUserClassLoader());
+		return new DefaultOperatorStateBackend(env.getUserClassLoader(), env.getExecutionConfig());
```

Comment:
It seems AbstractStateBackend.createOperatorStateBackend() already has an Environment parameter which has getExecutionConfig(), and The env comes form StreamTask. What do you think?